### ISSUE TITLE
Posted plants not syncing in mundraub.org

### ISSUE THREAD

**Issue by Porrumentzio** [niccokunzmann/mundraub-android#318]
Posted plants, though being saved on My plants, are not syncing to mundraub.org/map, so seemly not saving my posts for other users.
As default, in Connection settings is set to publish to the that web. But, as it is not asked, I have not Mundraub account.

Also, My plants are not shown on my own map (#293)

**Comment by niccokunzmann** [comment]
Thanks for reporting these errors!

**Comment by Txopi** [comment]
Do you need logs or something similar to check the problem?

**Comment by niccokunzmann** [comment]
No, I will not check at the moment as I have other things to do. It should be reproducible without logs.

**Comment by Txopi** [comment]
Hi. Any news about this?

**Comment by niccokunzmann** [comment]
Hi, new should be linked here so people are notified. I did not work on 
this any further. If someone would like to give it some attention, I am 
here to help.

**Comment by danilo-jlle** [comment]
Hi. In my case I cannot login at Mundraub, not even the website because it says my password is wrong. So I reset it, log out and the same thing happens again.
